mapper/thin_edge_json/src/json.rs

```diff
@@ -470,34 +450,12 @@ mod tests {
         assert_eq!(output.timestamp, Some(timestamp));
     }
 
-    #[test]
-    fn thin_edge_json_reject_invalid_utf8() {
-        let input = b"temperature\xc3\x28";
-
-        let expected_error =
-            r#"Invalid UTF8: invalid utf-8 sequence of 1 bytes from index 11: temperature..."#;
-        let output = ThinEdgeJson::from_utf8(input);
-
-        let error = output.unwrap_err();
-        assert_eq!(expected_error, error.to_string());
-    }
-
-    #[test]
-    fn thin_edge_json_reject_non_utf8_input() {
-        let input = b"\xc3\x28";
-
-        let expected_error = r#"Invalid UTF8: invalid utf-8 sequence of 1 bytes from index 0: ..."#;
-        let output = ThinEdgeJson::from_utf8(input);
-        let error = output.unwrap_err();
-        assert_eq!(expected_error, error.to_string());
-    }
```

Comment:
I would not remove these tests and be sure that the error messages are unchanged even if triggered by a different piece of code.